test/LibraryManager.IntegrationTest/FileSaveRestoreTests.cs

```diff
@@ -52,25 +52,15 @@ public void FileSaveRestore_AddDeleteLibrary()
     }
   ]
 }";
-            string deletingLibraryContent = @"{
-  ""version"": ""1.0"",
-  ""defaultProvider"": ""cdnjs"",
-  ""libraries"": []
-}";
 
             ReplaceFileContent(addingLibraryContent);
             Helpers.FileIO.WaitForRestoredFiles(pathToLibrary, expectedFiles, caseInsensitive: true);
-
-            ReplaceFileContent(deletingLibraryContent);
-            Helpers.FileIO.WaitForDeletedFiles(pathToLibrary, expectedFiles, caseInsensitive: true);
-
-            ReplaceFileContent(_libmanFileContent);
```

Comment:
As discussed offline. This is to validate the uninstall libraries can work when changing the libman.json file. Can you add it back?

Reply:
Add the logic back in iteration 2

---
In reply to: [208039258](https://github.com/aspnet/LibraryManager/pull/341#discussion_r208039258) [](ancestors = 208039258)